java/java.file.launcher/src/org/netbeans/modules/java/file/launcher/queries/MultiSourceRootProvider.java

```diff
@@ -67,9 +67,12 @@
     @ServiceProvider(service=ClassPathProvider.class, position=9_999),
     @ServiceProvider(service=MultiSourceRootProvider.class)
 })
+@Messages({
+    "SETTING_EnableMultiSourceRoot=false"
+})
 public class MultiSourceRootProvider implements ClassPathProvider {
 
-    public static boolean DISABLE_MULTI_SOURCE_ROOT = false;
+    public static boolean DISABLE_MULTI_SOURCE_ROOT = !"true".equals(Bundle.SETTING_EnableMultiSourceRoot());
```

Comment:
@matthiasblaesing this is the NB default which is inverted false -> disable is set to true.

the property is the branding for the LSP server which I suppose is for the VSCode use case (which enables it by default).

this is how I understand this at least - I could be wrong.

Reply:
Ah crap - yeah - overlooked that

Reply:
Does this work correctly if someone translates the messages?
Should we also add `"true"` to `@Messages`?

Reply:
@junichi11 not everything which is in property files is a user facing String. This is just a field which can be `true` or `false` - thats it. This is not going to be translated.

`@Messages` is just a mechanism to pull some properties into the code, since maintaining property files can be error prone.